ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/class-maps.yml

```
@@ -27,6 +27,19 @@ class_maps:
       vlan: 200-400
     - name: COS_RANGE
       vlan: 1-3
+    - name: DSCP_TEST_1
+      dscp: af11
+      ecn: ect-ce
+    - name: DSCP_TEST_2
+      dscp: 2-4, 6
```

```suggestion
      dscp: 2-4,6
```
Although accepted - EOS strips all whitespaces inside range definition